test/common/runtime/runtime_impl_test.cc

```diff
@@ -35,12 +36,30 @@ TEST(UUID, sanityCheckOfUniqueness) {
 
 class RuntimeImplTest : public testing::Test {
 public:
+  void SetUp() override {
+    // Some ugly ::system escaping. This is needed as this test relies on
+    // symlinks which can only exist in the test temporary directory to play
+    // nice with Bazel, so we create them on-the-fly. Could do this cleanly with
+    // std::filesystem if/when we have C++17.
+    ASSERT_EQ(0, ::system(("cp -rf --parents test/common/runtime/test_data " +
```

Comment:
I want to just clarify something for my understanding: If I read the code correctly, there is nothing that requires that the test dirs get cleaned between runs, right? Meaning, in bazel they might, but on my local machine they won't potentially (likely). So tests need to assume that there are previous artifacts? Is that the way to go? Or should we somehow require total cleanup between runs?